layers/+lang/c-c++/funcs.el

```diff
@@ -312,10 +303,9 @@ and the arguments for flyckeck-clang based on a project-specific text file."
     "hb" 'base) ;;Replace this with lsp-goto-implementation in lsp-layer?
 
   (when (eq c-c++-backend 'lsp-ccls)
-    (spacemacs/set-leader-keys-for-major-mode mode
-      "bR" 'ccls-reload)
+    ;; (spacemacs/set-leader-keys-for-major-mode mode
     (spacemacs/lsp-bind-extensions-for-mode mode "c-c++"
-      "mc" 'member-classes
```

Comment:
Also, you could add ccls-specific `"hd" 'derived` to match `"hb" 'base` function. Just need to add `(spacemacs/lsp-define-extensions "c-c++" 'derived "$ccls/inheritance" '(:derived t))` where custom extensions are defined